### PR TITLE
Beethoven: Implement non-ERC20 tokens collector

### PR DESCRIPTION
**Pull request by lgalende**

### CODE REVIEW

File: packages/users/beethoven/fee-collector/contracts/NonERC20BalancerClaimer.sol [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    function _beforeBalancerClaimer(address token, uint256 amount) internal override {
+        super._beforeBalancerClaimer(token, amount);
+        balanceBefore = IERC20(token).balanceOf(smartVault);
+    }
+
+    /**
+     * @dev After balancer claimer task hook
+     */
+    function _afterBalancerClaimer(address token, uint256) internal override {
+        uint256 balanceAfter = IERC20(token).balanceOf(smartVault);
```
Comment (lgalende) [inline packages/users/beethoven/fee-collector/contracts/NonERC20BalancerClaimer.sol:90]: Is it ok assuming `token` is the same as the one in the before hook?

Comment (facuspagnuolo) [reply packages/users/beethoven/fee-collector/contracts/NonERC20BalancerClaimer.sol:90]: It's fine

File: packages/users/beethoven/fee-collector/contracts/NonERC20BalancerClaimer.sol [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        uint256 amount = balanceAfter - balanceBefore;
```
Comment (facuspagnuolo) [inline packages/users/beethoven/fee-collector/contracts/NonERC20BalancerClaimer.sol:91]: We should set `balanceBefore` to zero here

File: packages/users/beethoven/fee-collector/contracts/NonERC20BalancerClaimer.sol (hunk 1 above)
Comment (facuspagnuolo) [reply packages/users/beethoven/fee-collector/contracts/NonERC20BalancerClaimer.sol:90]: It's fine

**Review by dmf7z — APPROVED** [review]

**Pull request merged**